### ISSUE TITLE
Install the default theme with pelican-quickstart

### ISSUE THREAD

**Issue by liltinkerer** [getpelican/pelican#2076]
Currently, pelican-quickstart doesn't install a theme by default.

From a beginners perspective, it would make more sense if the default theme (that is used anyway) is automatically added to themes folder and `THEME` variable set in the config. It would save a lot of hassle when a user tries to tweak few CSS settings and get to know how Pelican works.

**Comment by justinmayer** [comment]
Pelican is designed to use the built-in `notmyidea` theme by default. Given that we'll most likely switch to standard Python packaging for themes and plugins in the future, I'm not sure it makes sense to change how we're handling this at the moment.

**Comment by liltinkerer** [comment]
By standard packaging, do you mean pip?
Maybe it would make sense if that is done, pelican-quickstart downloads using pip the default (notmyidea) theme and then adds the `THEME` variable?

**Comment by justinmayer** [comment]
By standard Python packaging, I refer to PyPI, Pip, Setuptools, et cetera. c.f., #1564 #1655 https://github.com/getpelican/pelican-plugins/issues/425

As it stands now, the default theme is already available via Quickstart without having to download anything or configure any settings. My stance is that if someone is savvy enough to tinker with theme structure/CSS, then they aren't going to have any difficulty retrieving a theme folder and adding a one-line `THEME` variable. Plus, who's to say that someone who wants to tweak a theme wants to start with the default theme instead of one of the many others?

Assuming we decide to switch to Python packaging for theme/plugin installation, the canonical installation docs would likely change to `pip install pelican your-fav-theme`, at which point Pelican could default to that theme if it is the only one installed.

But even that scenario raises other questions. How would your tinkerer modify the theme in this latter scenario? By editing the installed theme directly in site-packages? That seems like a potentially sub-standard approach.

In summary, I don't know how this will be handled going forward, but so far I'm not sensing sufficient reason to change how the default theme is handled until the longer-term plan is sorted out.
